content/06.know-your-problem.md

```diff
@@ -1,41 +1,40 @@
 ## Tip 4: Know your data and your question {#know-your-problem}
 
 Having a well defined scientific question and a clear analysis plan is crucial for carrying out a successful deep learning project.
-Just like it would be inadvisable to step foot in a laboratory and begin experiments without having a defined endpoint, a deep learning project should not be undertaken without preparation.
-Foremost, it is important to assess if a dataset exists that can answer the biological question of interest for the given deep learning model; obtaining said data and associated metadata and reviewing the study protocol should be pursued as early on in the project as possible.
-A publication or resource might purportedly offer a dataset that seems to be a good fit to test your hypothesis, but the act of obtaining it can reveal numerous problems.
-It may be unstructured when it is supposed to be structured, crucial metadata such as sample stratification are missing, or the usable sample size is different than what is reported.
-Data collection should be documented or a data collection protocol should be created and specified in the project documentation.
-Information such as the resource used, the date downloaded, and the version of the dataset, if any, will help minimize operational confusion and will allow for transparency during the publication process.
+Just like it would be inadvisable to set foot in a laboratory and begin experiments without having a defined endpoint, a deep learning project should not be undertaken without defined goals.
+Foremost, it is important to assess if a dataset exists that can answer the biological question of interest using a deep learning-based approach.
+If so, obtaining this data (and associated metadata), and reviewing the study protocol, should be pursued as early on in the project as possible.
+This can help to ensure that data is as expected, and can prevent the wasted time and effort that occur when issues are discovered later on in the analytic process.
+For example, a publication or resource might purportedly offer an appropriate dataset that is found to be inadequate upon aquisition.
+The data may be unstructured when it is supposed to be structured, crucial metadata such as sample stratification might be missing, or the usable sample size may be different than expected.
+Any of these data issues might limit a researcher's ability to use DL to address the biological question at hand, or might otherwise require adjustment before DL can be used.
+Data collection should also be carefully documented, or a data collection protocol should be created and specified in the project documentation.
+Information about the resources used, downloaded dates, and dataset versions will help to minimize operational confusion, and will increase transparency and reproducibility during publication and beyond.

```

Comment:
```suggestion
Information about the resources used, downloaded dates, and dataset versions will help to minimize operational confusion and will increase transparency and reproducibility during publication and beyond.
```

Reply:
I'd suggest to keep this comma - there are just SO many "and"s

Reply:
I think this sentence just needs to be fundamentally revised. I agree there are too many ands.

Reply:
I love how the syntax highlighter shows all the "and"s. How about:

```suggestion
Information about the resources used, download dates, and dataset versions are critical to preserve.
Doing so will help to minimize operational confusion and will increase the reproducibility of the analysis.
```